src/Groups/perm.jl

```diff
@@ -161,6 +161,13 @@ julia> length(moved_points(gen(s, 1)))
 ```
 """
 @gapattribute moved_points(x::Union{PermGroupElem,PermGroup}) = Vector{Int}(GAP.Globals.MovedPoints(GapObj(x)))
+"""
```

Comment:
```suggestion

"""
```
Please add a newline in between two functions